### PR TITLE
File Manager - Provide the onSelectedFileOpened event

### PR DESCRIPTION
**Pull request by maxerokh**

### CODE REVIEW

File: js/ui/file_manager/ui.file_manager.js [hunk 1]
```diff
@@ -453,6 +454,17 @@ class FileManager extends Widget {
             */
             customizeDetailColumns: null,
 
+            /**
+            * @name dxFileManagerOptions.onSelectedFileOpened
+            * @extends Action
+            * @type function(e)
+            * @type_function_param1 e: object
+            * @type_function_param1_field1 fileItem:object
```
Comment (RomanTsukanov) [inline js/ui/file_manager/ui.file_manager.js]:
```suggestion
            * @type_function_param1_field4 fileItem:object
```

The Action doccomment contains definitions for three fields that are always present in an action. Additional fields are numbered beginning with 4.

File: js/ui/file_manager/ui.file_manager.js [hunk 2]
```diff
@@ -453,6 +454,17 @@ class FileManager extends Widget {
             */
             customizeDetailColumns: null,
 
+            /**
+            * @name dxFileManagerOptions.onSelectedFileOpened
+            * @extends Action
+            * @type function(e)
+            * @type_function_param1 e: object
```
Comment (RomanTsukanov) [inline js/ui/file_manager/ui.file_manager.js]:
```suggestion
            * @type_function_param1 e:object
```

**Review by RomanTsukanov — DISMISSED** [review]

File: js/ui/file_manager/ui.file_manager.js [hunk 3]
```diff
@@ -127,6 +127,7 @@ class FileManager extends Widget {
             onError: ({ error }) => this._showError(error),
             onSelectionChanged: this._onItemViewSelectionChanged.bind(this),
             onSelectedItemOpened: ({ item }) => this._tryOpen(item),
+            onSelectedFileOpened: this._createActionByOption("onSelectedFileOpened"),
```
Comment (vladkovl) [inline js/ui/file_manager/ui.file_manager.js:132]: We can use existing `onSelectedItemOpened `option for this task

File: js/ui/file_manager/ui.file_manager.js (hunk 1 above)
Comment (maxerokh) [reply js/ui/file_manager/ui.file_manager.js]: Repaired

File: js/ui/file_manager/ui.file_manager.js (hunk 3 above)
Comment (maxerokh) [reply js/ui/file_manager/ui.file_manager.js:132]: repaired

**Review by vladkovl — DISMISSED** [review]

**Review by vladkovl — APPROVED** [review]

**Pull request merged**